cobridge_base/include/websocket_server.hpp

```diff
@@ -131,8 +132,6 @@ status_level_to_log_level(StatusLevel level)
       return APP;
     case StatusLevel::Warning:
       return WARNING;
-//            case StatusLevel::Error:
```

Comment:
what if the status is error? After this change, it seems we cannot handle this type of error.